web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -347,6 +347,9 @@ const MoreCast2Page = () => {
     }
   }
 
+  const displayForecastRows = [...forecastRows]
+  displayForecastRows.sort((a, b) => (a.forDate > b.forDate ? 1 : -1))
```

Comment:
The deployment worked! Sorting by date will give us something like this screenshot when more than one station is selected.
![image](https://user-images.githubusercontent.com/7094320/227339827-ab04d9d6-998f-4bd3-bcb5-fc1827a56ee8.png)

Is this what we want?